File: Modul 4/C/PRAK405.c
#include <stdio.h>
int main() {
    printf("Input\n");
    int angka1, angka2, x, y, total;
    scanf("%d %d", &angka1, &angka2);
    printf("Output\n");

    for (int i = 1; i <= angka1; i++) {
        int a = i;
        while (a > 1) {
            printf("(%d * %d) + ", a, angka2);
            a--;
        }
        int b = 1;
        int hasil = i * angka2;
        while (b < i) {
            hasil += b * angka2; //menyimpan hasil operasi untuk setiap iterasi
            b++;
        }
        printf("(%d * %d) = %d\n", a, angka2, hasil);
    }

    // Menghitung hasil akhir nilai dari penjumlahan
    x = 1;
    y = 1;
    total = 0;
    while (x <= angka1) {
        total += y * angka2;
        y += x + 1;
        x++;
    }
    printf("%d\n", total);
    return 0;
}

File: Modul 4/PYTHON/PRAK405.py
print("Input")
angka1, angka2 = input().split()
angka1 = int(angka1)
angka2 = int(angka2)
print("Output")
for i in range(1, angka1 + 1):
    a = i 
    while a > 1 : 
        print("(%d * %d) + " %(a, angka2), end='') 
        a -= 1 
    b = 1 
    hasil = (i*angka2)
    while b < i:
        hasil += (b * angka2) 
        b += 1
    print("(%d * %d) = %d" %(a, angka2, hasil))

x = 1
y = 1
total = 0
while x <= angka1:
    total += y * angka2 
    y += x + 1
    x += 1
print("%d" %(total))